cmd/integration/integration.go

```diff
@@ -47,10 +47,10 @@ import (
 	"k8s.io/kubernetes/pkg/controller/node"
 	replicationControllerPkg "k8s.io/kubernetes/pkg/controller/replication"
 	"k8s.io/kubernetes/pkg/fields"
-	"k8s.io/kubernetes/pkg/kubelet"
 	"k8s.io/kubernetes/pkg/kubelet/cadvisor"
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
 	"k8s.io/kubernetes/pkg/kubelet/dockertools"
+	kubeletTypes "k8s.io/kubernetes/pkg/kubelet/types"
```

Comment:
I know this is consistent with what we already have, but what do you think about `kubetypes` instead? It's shorter and consistent with `kubecontainer`. Probably makes sense to change separately.